lmdk/modules/smart_amp_test/smart_amp_test.c

```diff
@@ -0,0 +1,317 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
This needs to be in modules directory and we need to reuse code, we cant have 2 implementations of dummy smart amp. i.e. I would use Kconfig to select which functions to build in library mode and which to build in as I imagine most of the code will be the same.